src/loader.ts

```diff
@@ -111,9 +119,9 @@ export const resolve: Resolve = async function (ident, context, fallback) {
 	}
 
 	return fallback(ident, context, fallback);
-}
+})
 
-export const load: Load = async function (uri, context, fallback) {
+export const load: Load = withShortCircuit(async function (uri, context, fallback) {
```

Comment:
Can you remove the wrapper function & just add `shortCircuit: true` to the `return`ed object literals; eg https://github.com/lukeed/tsm/blob/44f9108f95b8554348fb6602d07aea47081edf0b/src/loader.ts#L92

```suggestion
export const load: Load = async function (uri, context, fallback) {
```

Reply:
Done